### ISSUE TITLE
[Request] Network Profiles, like which AdGuard options would be enabled/disabled for each WiFi network or mobile data

### ISSUE THREAD

**Issue by remewer** [AdguardTeam/AdguardForAndroid#1931]
For example, my company's WiFi network, they don't allow third party DNS and VPN servers. So I have to disable DNS and Proxy in AdGuard every time I connect to my company's network. However, when I'm not at work and I'm using my mobile data, I want those to to be turned on.

Other options that may be toggled other than DNS and Proxy depending on network, are Filtering Mode, Filter Lists, HTTPS Filtering, apps to block/unblock for a specific network, etc., you name it.

Or at least open up those options to Tasker, who'll take care of the "which network" part.

**Comment by ameshkov** [comment]
> Or at least open up those options to Tasker, who'll take care of the "which network" part.

This will be handled in https://github.com/AdguardTeam/AdguardForAndroid/issues/1896

**Comment by Eugene-Savenko** [comment]
A similar request by 2277949

> To be able to quickly apply profiles. I want to allow ads for a specific app. However, to be able to see ads I must allow additional apps (Play Services, Google framework...). When I don't use this specific I'd like to block ads in all apps. Today I have to do it manually so I'd like to know if there's a tip to do it automatically or if you could implement a feature to manage mass modifications.

**Comment by ameshkov** [comment]
Merged into https://github.com/AdguardTeam/AdguardForAndroid/issues/3379
